_data/software/net-core.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      .NET Core is a free and open-source, managed computer software framework
+      that is a cross-platform successor to .NET Framework.
+    fr: >-
+      .NET Core est un framework de logiciel informatique gratuit et open
+      source, qui est un successeur multiplateforme de .NET Framework.
+category: '43232400'
+homepageURL:
+  en: 'https://dotnet.microsoft.com/'
+  fr: 'https://dotnet.microsoft.com/'
```

Comment:
I was surprised they don't have a localized main site, just the docs I guess

Reply:
Me too. :(